sdk/python/test_picovoice.py

```diff
@@ -21,89 +18,101 @@
 
 from picovoice import Picovoice
 
+from test_util import *
 
-class PicovoiceTestCase(unittest.TestCase):
-    @staticmethod
-    def _context_path():
-        if platform.system() == 'Darwin':
-            return os.path.join(
-                os.path.dirname(__file__),
-                '../../resources/rhino/resources/contexts/mac/coffee_maker_mac.rhn')
-        elif platform.system() == 'Linux':
-            if platform.machine() == 'x86_64':
-                return os.path.join(
-                    os.path.dirname(__file__),
-                    '../../resources/rhino/resources/contexts/linux/coffee_maker_linux.rhn')
-            else:
-                cpu_info = ''
-                try:
-                    cpu_info = subprocess.check_output(['cat', '/proc/cpuinfo']).decode()
-                    cpu_part_list = [x for x in cpu_info.split('\n') if 'CPU part' in x]
-                    cpu_part = cpu_part_list[0].split(' ')[-1].lower()
-                except Exception as error:
-                    raise RuntimeError("Failed to identify the CPU with '%s'\nCPU info: %s" % (error, cpu_info))
-
-                if '0xb76' == cpu_part or '0xc07' == cpu_part or '0xd03' == cpu_part or '0xd08' == cpu_part:
-                    return os.path.join(
-                        os.path.dirname(__file__),
-                        '../../resources/rhino/resources/contexts/raspberry-pi/coffee_maker_raspberry-pi.rhn')
-                elif '0xd07' == cpu_part:
-                    return os.path.join(os.path.dirname(__file__),
-                                        '../../resources/rhino/resources/contexts/jetson/coffee_maker_jetson.rhn')
-                elif '0xc08' == cpu_part:
-                    return os.path.join(
-                        os.path.dirname(__file__),
-                        '../../resources/rhino/resources/contexts/beaglebone/coffee_maker_beaglebone.rhn')
-                else:
-                    raise NotImplementedError("Unsupported CPU: '%s'." % cpu_part)
-        elif platform.system() == 'Windows':
-            return os.path.join(os.path.dirname(__file__), '../../resources/rhino/resources/contexts/windows/coffee_maker_windows.rhn')
-        else:
-            raise ValueError("Unsupported system '%s'." % platform.system())
 
-    @classmethod
-    def _wake_word_callback(cls):
-        cls._is_wake_word_detected = True
+class PicovoiceMetaData:
```

Comment:
You could add an instance of `Picovoice` as a property of this class (`self.picovoice`) so that you don't have to keep fetching it as a tuple. Then maybe the name `PicovoiceTestData` would be more fitting.